src/plugins/controls/common/control_group/control_group_panel_diff_system.ts

```diff
@@ -26,6 +27,19 @@ export const genericControlPanelDiffSystem: DiffSystem = {
 export const ControlPanelDiffSystems: {
   [key: string]: DiffSystem;
 } = {
+  [RANGE_SLIDER_CONTROL]: {
+    getPanelIsEqual: (initialInput, newInput) => {
+      if (!deepEqual(omit(initialInput, 'explicitInput'), omit(newInput, 'explicitInput'))) {
+        return false;
+      }
+
+      const { value: valueA = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+        initialInput.explicitInput;
+      const { value: valueB = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+        newInput.explicitInput;
+      return isEqual(valueA, valueB);
+    },
+  },
```

Comment:
This fixes a bug where resetting the selections back to "undefined" / empty would cause unsaved changes:

**Before:**

**After:**